### ISSUE TITLE
Make ios_system available as a Swift Package

### ISSUE THREAD

**Issue by BrianChevalier** [holzschu/ios_system#84]
The Swift Package manager is now available directly in Xcode where package dependencies can be managed by version, and compiled for the appropriate platform. Packages can also specify their own dependencies (useful for python_ios, and libtex) so new developers don't have to manage that complexity. This will make it an even easier drop-in replacement for system() in iOS apps, as well as testing in the simulator and UIKit for Mac apps.

[Creating Swift Packages WWDC19 Session](https://developer.apple.com/videos/play/wwdc2019/410)

**Comment by BrianChevalier** [comment]
I'll be exploring converting ios_system to a Swift Package and adding updates/resources here.

**Comment by kkk669** [comment]
Isn't this issue already resolved?

**Comment by holzschu** [comment]
Yes, but PR #109 is related to that as well. It keeps the binary swift package system, but it changes the method of delivery.
Would you have feedback on the PR?

**Comment by kkk669** [comment]
@holzschu His proposal is great. For example, also in a-Shell, you can fetch XCFrameworks with `swift package --package-path xcfs resolve` and then their paths become relative to the project directory (e.g. `$PROJECT_DIR/xcfs/.build/artifacts/xcfs/*.xcframework`). That could resolve [this issue](https://github.com/holzschu/a-shell/issues/109) as well.

In the future, if SwiftPM supports embedded binary framework, you can go back to Xcode's built-in SwiftPM again.
